src/app/myworkflows/sidebar-accordion/github-apps-logs/github-apps-logs.component.ts

```diff
@@ -61,57 +51,44 @@ export class GithubAppsLogsComponent extends Base implements OnInit, AfterViewIn
   private pageFilter: string;
 
   isExpansionDetailRow = (i: number, row: Object) => row.hasOwnProperty('detailRow');
+  public dataSource: LambdaEventDataSource;
 
   constructor(
     @Inject(MAT_DIALOG_DATA) public matDialogData: { userId?: number; organization?: string },
     private paginatorService: PaginatorService,
     private lambdaEventsService: LambdaEventsService,
-    private matSnackBar: MatSnackBar,
-    private mapPipe: MapFriendlyValuesPipe,
-    private paginatorQuery: PaginatorQuery
+    private alertService: AlertService
   ) {
     super();
-    this.datePipe = new DatePipe('en');
-    const defaultPredicate = this.dataSource.filterPredicate;
-    this.dataSource.filterPredicate = (data, filter) => {
```

Comment:
This code wasn't actually effectively doing anything as far as I could tell. It's for client-side filtering, but before it's triggered, the server-side pagination triggers first, so anything this could filter would get removed, e.g., type "Success", and no records come back from the API.